database.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * CS 4241 Final Project - Database component
+ * by Terry Hearst, Demi Karavoussianis, Kyle Reese, and Tom White
+ */
+
+ const http = require( 'http' ),
+ Sequalize = require('sequelize'),
+      // IMPORTANT: you must run `npm install` in the directory for this assignment
+      // to install the mime library used in the following line of code
+      mime = require( 'mime' ),
+      dir  = 'public/',
+      port = 3000
+
+// Simple file based db using an ORM
+const sequelize = new Sequalize({
+  dialect: 'sqlite',
+  storage: 'database.lite',
```

Comment:
Make sure that you  `npm install` any libraries you need to get this working with SQLite. I don't think it comes with out of the box support.